src/StagesCards.js

```diff
@@ -1,8 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './styles/App.css';
 
 
-
+class StagesCards extends Component {
+  constructor() {
+    super();
+    this.state = {
```

Comment:
Since we are just passing props through to this class and aren't using state, the constructor method with the state isn't being used.  You can get rid of it and it shouldn't affect the code